### PR TITLE
Move from webpacker to vite

### PR DESCRIPTION
**Pull request by hackartisan**
Trying this as a solution to a ci (and local development) failure on the node 18 upgrade - see https://app.circleci.com/pipelines/github/pulibrary/pulfalight/2556/workflows/1ea3eb1c-c29f-415b-b173-d1d6c5115fa2/jobs/8291 for failure details. 

Seemed good to do it regardless, for consistency with our other projects.

### CODE REVIEW

**Comment by hackartisan** [comment]
This is deployed to staging and qa

File: app/javascript/entrypoints/application.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// To see this message, add the following to the `<head>` section in your
+// views/layouts/application.html.erb
+//
+//    <%= vite_client_tag %>
+//    <%= vite_javascript_tag 'application' %>
+console.log('Vite ⚡️ Rails')
```
Comment (eliotjordan) [inline app/javascript/entrypoints/application.js:6]: suggestion: we should probably remove console.log statements.

File: app/javascript/entrypoints/application.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// To see this message, add the following to the `<head>` section in your
+// views/layouts/application.html.erb
+//
+//    <%= vite_client_tag %>
+//    <%= vite_javascript_tag 'application' %>
```
Comment (eliotjordan) [inline app/javascript/entrypoints/application.js:5]: suggestion: we don't need this comment as we've already added these tags to application.html.erb.

**Review by eliotjordan — CHANGES_REQUESTED** [review]
Looks good. Just a couple of comments.

**Review by eliotjordan — APPROVED** [review]

**Pull request merged**